pkg/cli/cli.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+// Description: See package comment
```

Comment:
This is for the file contents, isn't it? This package has multiple files in it - shouldn't this describe the contents of this file?

Reply:
It is, but the package comment describes what's in this file. I can duplicate it if you'd prefer

Reply:
Shouldn't you fix the package comment, then?

Reply:
Yeah, I guess it was slightly messed up. Should be better now. LMK